crates/agglayer-node/src/rate_limiting/local/trivial.rs

```diff
@@ -0,0 +1,50 @@
+/// Rate limiter handling edge cases.
+///
+/// This rate limiter handles edge cases, namely if no rate limit is imposed or
+/// if rate limit is zero. Otherwise, it delegates to the inner limiter.
+pub enum RateLimiter<L> {
+    Disabled,
+    Limited(L),
+    Unlimited,
```

Comment:
Could we have some explanation of the different variants ?
Such as `Disabled` mean that its an error and that the RPC is disabled?